environments/21.08/spack.yaml

```diff
@@ -0,0 +1,327 @@
+spack:
+
+  view: false
+  concretization: separately
+
+  packages:
+    all:
+      compiler:
+        - gcc@9.3.0
+      providers:
+        blas:
+          - openblas
+        mpi:
+          - mpich
+      target:
+        - x86_64
```

Comment:
it could be easier to put them in one line it is easier to read as pose to reading multiline list 
```
compiler: [gcc@9.3.0]
providers:
  blas: [openblas]
  mpi: [mpich]
  target: [x86_64]
```

Reply:
It is perfectly readable as it is. If you would like to have these changes, please feel free.